src/libraries/Microsoft.PowerFx.Transport.Attributes/Microsoft.PowerFx.Transport.Attributes.csproj

```diff
@@ -33,7 +33,7 @@
   <!-- Nuget Properties -->
   <PropertyGroup>
     <PackageId>Microsoft.PowerFx.Transport.Attributes</PackageId>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>    
```

Comment:
Nit: looks like a bit of extra white space was added in all these csproj files. 
run git checkout main — ‘*.csproj’ to clean that up